tests/dist-check.py

```diff
@@ -0,0 +1,59 @@
+"""
+Integration test to check the integrity of the
+distribution.
+
+This file is explicitly not named 'test_' to avoid
+being collected by pytest, but must instead be
+invoked explicitly (i.e. pytest tests/dist-check.py).
+
+This test cannot be invoked as part of the normal
+test suite nor can it be included in the normal test
+suite because it must import cherrypy late (after
+removing sys.path[0]).
+"""
+
+import os
+import sys
+
+import pytest
+
+
+@pytest.fixture(params=[
+    'favicon.ico',
+    'scaffold/static/made_with_cherrypy_small.png',
+    'tutorial/tutorial.conf',
+    'tutorial/custom_error.html',
+])
+def data_file_path(request):
+    "generates data file paths expected to be found in the package"
+    return request.param
+
+
+@pytest.fixture(autouse=True, scope="session")
+def remove_sys_path_0():
+    "pytest adds cwd to sys.path[0]"
+    print("removing", sys.path[0])
+    del sys.path[0]
+    assert 'cherrypy' not in sys.modules
+
+
+def test_data_files_installed(data_file_path):
+    """
+    Ensure that data file paths are available in the
+    installed package as expected.
+    """
+    import cherrypy
+    root = os.path.dirname(cherrypy.__file__)
+    fn = os.path.join(root, data_file_path)
```

Comment:
`fn` is a perfectly fine variable name for a test like this. I can't imagine how to disable this check in a way that's not even more messy or removing the check entirely. I'm not going to worry about it now. It's not worth my time.